src/components/quickStarts.stories.mdx

```diff
@@ -484,4 +426,216 @@ will automatically stop any video processor that was previously running.
 
 You may also refer to the storybook documentation for other examples on how to use [`BackgroundBlurProvider`](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/docs/sdk-providers-backgroundblurprovider--page) and `createBackgroundBlurDevice`.
 
-For more documentation on creating a `VideoTransformDevice`, you can refer to the [Background Filter Video Processor Guide](https://github.com/aws/amazon-chime-sdk-js/blob/master/guides/15_Background_Filter_Video_Processor.md).
\ No newline at end of file
+For more documentation on creating a `VideoTransformDevice`, you can refer to the [Background Filter Video Processor Guide](https://github.com/aws/amazon-chime-sdk-js/blob/master/guides/15_Background_Filter_Video_Processor.md).
+
+## Adding Attendee Names to Build a Roster
+
+Chime does not manage the names of your attendees. If you want video nameplates or a realtime roster, you need to provide a function to `meetingManager.getAttendee` that returns the name of the attendee for a given chime attendee ID and external user ID.
```

Comment:
You are also explaining video nameplates but the title only talks about the roster. So suggest adding video nameplates in the title as well.

"Chime does not include attendee names implicitly when building the meeting roster or the video grid. Provide a function to `meetingManager.getAttendee` that returns the name of the attendee for a given Chime AttendeeId and ExternalUserId."

Reply:
Changed the title to "Create a Callback Function to Get the Attendee Name" and rephrased the sentence.